R/flatten.R

```diff
@@ -24,36 +24,36 @@
 #' x %>% map_int(1L)
 #' @useDynLib purrr flatten_impl vflatten_impl
 flatten <- function(.x) {
-  .Call(flatten_impl, .x)
+  .Call(flatten_impl, .x, PACKAGE = "purrr")
```

Comment:
I think @kevinushey misread this - I'm using `flatten_impl` not "`flatten_impl`" so it doesn't need the package argument.